Contributor-dashboard.md

```diff
@@ -36,31 +36,23 @@ There exists a separate admin page for the contributor dashboard at /contributor
 See [this](https://docs.google.com/document/d/1VqNiJttq85YyR6cQkd8M9lGGkOP8OlUlkI37Xw6SovM/edit) doc for step-by-step admin instructions. This may be useful for developing locally as a coder as well.
 
 ## Local development
-Some setup is usually required when developing locally for the contributor dashboard since before a user can submit a content suggestion to a lesson, a lesson needs to exist. Additionally, the requirements outlined in [How items for contribution are populated](#how-items-for-contribution-are-populated) must be satisfied. To pre-populate contributor dashboard data, run the `populate_sample_contributor_data.py` script after starting a local development server:
+Some setup is usually required when developing locally for the contributor dashboard since before a user can submit a content suggestion to a lesson, a lesson needs to exist. Additionally, the requirements outlined in [How items for contribution are populated](#how-items-for-contribution-are-populated) must be satisfied. 
+
+To populate contributor dashboard data, first start your local server using:
 
 ```
 python -m scripts.start
 ```
 
-```
-python -m scripts.populate_sample_contributor_data
-```
+And then navigate to Admin page and go to the "Roles" tab. Assign yourself the "Curriculum Admin" role.
+Now go back to the "Activities" tab of Admin page and click on "Load Data" as shown in the below screenshot.
```

Comment:
of --> on the

Also perhaps add a blank line above this one.

Reply:
Done.